packages/key-management/src/util/createAddressManager.ts

```diff
@@ -0,0 +1,53 @@
+import * as Crypto from '@cardano-sdk/crypto';
+import {
+  AccountAddressDerivationPath,
+  AccountKeyDerivationPath,
+  AddressManager,
+  AsyncKeyAgent,
+  GroupedAddress
+} from '../types';
+import { Observable } from 'rxjs';
+
+/** An address manager that uses a {@link AsyncKeyAgent} to derive addresses. */
+export class Bip32Ed25519AddressManager implements AddressManager {
```

Comment:
`AdressManager` looks like a duplicate of `AsyncKeyAgent` or at least it's just using the same methods of the passed key agent?